ui/appui-react/src/appui-react/statusfields/SelectionCount.scss

```diff
@@ -6,9 +6,22 @@
   gap: var(--iui-size-2xs);
   display: flex;
   align-items: center;
-  padding: 0 var(--iui-size-2xs) 0 var(--iui-size-2xs);
+  min-width: var(--iui-size-2xl);
+  padding-left: var(--iui-size-2xs);
 
   .icon {
     color: var(--iui-color-icon);
   }
 }
+
+.uifw-statusBar-right
```

Comment:
Looks good for a quick fix, but ideally the component should add a class name for these conditions (i.e. you could wrap each item in React context with needed info). Please add some screenshots as well.

Reply:
Screenshots added. I suppose you don't want me to add the context now?

Reply:
If you could, I'd prefer to do it now